crontab/notify_old_pp.php

```diff
@@ -5,98 +5,107 @@
-If you are waiting on missing images or page scans, please add the details to the Missing Page Wiki at: $forums_url/viewtopic.php?t=7584\n\n
-If you no longer wish to have this text assigned to you please visit the $site_name website Post Processing section and select Return to Available for this book, or forward this email to $general_help_email_addr and state that you would no longer like to have the book in question assigned to you so that we may return it to the available pool for someone else to work on.\n\n
-$site_signoff";
-            } else {
-                $message = "Hello $PPinQuestion,\n\nThis is an automated message.\n\n
-Our database indicates that you have had $numprojs PP projects checked out for more than 90 days:\n\n
-$projectslist\n\n
-If you wish to continue working on some or all of these books, please log in to $site_url and visit each such project's home-page (copy the URL listed with the project above and paste it into your browser's address-field). Doing this will update the status of the project and let us know that you are still working on it. If you need help please forward this email, quoting the list of books, with a brief description of the status for each of the various books listed above that you need help with to $general_help_email_addr.\n\n
-If you have completed your work on any of these books, please log in to $site_url and visit each such project's home-page (copy the URL listed with the project above and paste it into your browser's address-field). Select the 'Upload for verification' option and follow the prompts. You will be able to leave a message for the verifier during this process, if you have any special information or comments to pass on.\n\n
-If you are waiting on missing images or page scans, please add the details to the Missing Page Wiki at: $forums_url/viewtopic.php?t=7584\n\n
-If you no longer wish to have some or all of these books assigned to you please visit the $site_name website Post Processing section and select Return to Available for the books in question or forward this email to $general_help_email_addr and state that you would no longer like to have the books in question assigned to you so that we may return them to the available pool for someone else to work on.\n\n
-$site_signoff";
-            }
-
-            maybe_mail("$email", "$subject","$message");
-
-            $projectslist = "";
-            $numprojs = 0;
-         }
-        $PPinQuestion = $checkedoutby;
-    }
+$projects_by_PPer = get_pp_projects_past_threshold();
 
-    $numprojs++;
+// if the script is run before the 14th of the month, send out the first email
+// otherwise send out the second one
+if(strftime("%d", time()) < 14)
+{
+    $which_email = "first";
```

Comment:
"email" is sometimes a shorthand for "email address", so `$which_message` might be clearer.